migrations/migrations/20191025184730-removeTimestampsFromUser.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Why did you remove the timestamps?

Reply:
Cause they were disabled. I will try keep them.